### PR TITLE
Babel Upgrade

### PR DESCRIPTION
**Pull request by marciliojrs**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```
Comment (lunks) [inline .vscode/launch.json]: JSON doesn't have comments (or it shouldn't 😂 )

File: babel.config.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module.exports = function(api) {
+  api.cache(true);
```
Comment (lunks) [inline babel.config.js]: Do you need this? Not sure.

File: .vscode/launch.json (hunk 1 above)
Comment (marciliojrs) [reply .vscode/launch.json]: hahaha VS generated files 🤦‍♂️

File: babel.config.js (hunk 2 above)
Comment (marciliojrs) [reply babel.config.js]: probably not, I got this from Babel docs

**Pull request merged**